tests/testfile.txt

```diff
@@ -0,0 +1 @@
+lalala
```

Comment:
What's this?

Reply:
This file used to test if `/files` endpoint works: https://github.com/vrslev/frappe_docker/blob/5e16ed89b6695eab7b486164c2e13b5cdf750de8/tests/main.py#L223